sample-plugin/pom.xml

```diff
@@ -296,6 +301,16 @@
       <artifactId>json-path-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>mariadb-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>mysql-api</artifactId>
```

Comment:
Is this not implied by `database-mysql`?

Reply:
Yes but also some plugin like https://plugins.jenkins.io/pipeline-maven-database/dependencies/ are not using (yet) database plugin.

h2 and postgres API where also added on https://github.com/jenkinsci/bom/pull/2655/files

Reply:
Maybe we are discussing different things. To clarify: we avoid listing plugins in `sample-plugin/pom.xml` which would be transitive dependencies anyway due to some higher-level plugin. Given https://github.com/jenkinsci/database-mysql-plugin/blob/a0596d2b1438173b602ada90455094b04e7066e4/pom.xml#L47-L51 it should suffice to list only `database-mysql`.